Dockerfile

```diff
@@ -9,20 +11,21 @@ RUN apk add --no-cache gcc musl-dev git && \
 
 WORKDIR /go/src/github.com/joohoi/acme-dns
 
-RUN CGO_ENABLED=1 go build -ldflags="-s -w -linkmode external -extldflags -static"
+RUN CGO_ENABLED=1 go build
 
 RUN install -d -o 1000 -g 0 -m 0775 /build/etc/acme-dns /build/var/lib/acme-dns && \
-    install -o 1000 -g 0 -m 0554 /go/src/github.com/joohoi/acme-dns/acme-dns /build/acme-dns && \
+    install -D -o 1000 -g 0 -m 0554 /go/src/github.com/joohoi/acme-dns/acme-dns /build/usr/local/bin/acme-dns && \
     install -D -o 1000 -g 0 -m 0664 /go/src/github.com/joohoi/acme-dns/config.cfg /build/etc/acme-dns/config.cfg && \
     sed -Ei -e "s/(port ?= ?)\"80\"/\1\"8080\"/" \
             -e "s/(listen ?= ?)\".*:53\"/\1\":5353\"/" /build/etc/acme-dns/config.cfg
 
 
-FROM scratch
+FROM alpine:${ALPINE_VERSION}
 
 LABEL maintainer="joona@kuori.org"
 COPY --from=builder /build /
 USER 1000:0
 EXPOSE 5353/udp 5353 8080 8443
+HEALTHCHECK --interval=5s --timeout=1s CMD wget --quiet --tries=1 --spider http://127.0.0.1:8080/health || exit 1
```

Comment:
Using `--timeout` here without `-T` for `wget` seems like a bad idea.

I'd suggest a higher timeout and interval along with:
```
wget --spider --quiet -ST 1 http://127.0.0.1:8080/health || exit 1
```

In the case of failure, it's useful to have the output for inspection with docker.

Reply:
Thanks, I will change it.